tsup.config.ts

```diff
@@ -55,13 +56,14 @@ const browserConfig: Options = {
   name: 'browser',
   platform: 'browser',
   entry: ['./src/browser/index.ts'],
-  external: [mswCore, ecosystemDependencies],
+  external: [ecosystemDependencies],
```

Comment:
Types are likely failing because we excluded `mswCore` here. We mustn't do that. If we do, we end up in `browser.d.ts` having _their own_ `RequestHandler` class than `msw/lib/core.d.ts`, and so this code won't compile:

```ts
import { http } from 'msw'
import { setupWorker } from 'msw/browser'
// These will think of different "RequestHandler" class!
```

Reply:
If we keep `mswCore` in `external`, it will compile correctly and the CI should pass. All while giving us the bundling for the _rest_ of the dependencies we do use also. Wouldn't that be the goal here?

Reply:
> All while giving us the bundling for the rest of the dependencies we do use also.

Yes, thats what I had initially, but I noticed it left internal imports in-tact, which seemed odd to me, and so I bundled those as well. I think for now bundling the external dependencies is fine indeed 👍